docs/command/query.soy

```diff
@@ -1641,7 +1641,7 @@ on which the <code>one</code> library depends.
 <p>
 <pre>
 {literal}
-$ buck query "deps(//examples:one)"
+$ buck query --json "deps(//examples:one)"
```

Comment:
Hi @mattfenwick, `--output-format json` is preferred option to display JSON representation. Could you rebase to master and make the change?

Reply:
done!!

thanks for taking a look!